docs/tests/jest.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started
+
+## Running a Jest test
+
+```bash
+> yarn jest <path>
+```
+
+## Debugging
+
+### Debugging Jest in VS Code
+
+1. Open **VS Code**
```

Comment:
hi @DDDDDanica, 

step #2 and #3 are VS Code specific. I think we could add other IDE instructions if we find them useful

Reply:
gotcha ! i was testing this in intelliJ and the steps are the same, that's why i was thinking it could be more general. But for now it is helpful and thanks a lot for the documentation !

Reply:
interesting! exactly the same? in that case, we can update this to include intelliJ too. I don't want to install intelliJ for this. Please feel free to push commits to this PR, or I can update the code to mention intelliJ too 

@DDDDDanica